src/sentry/integrations/vercel/integration.py

```diff
@@ -195,7 +195,13 @@ def get_organization_config(self):
                     "placeholder": _("Vercel project..."),
                 },
                 "sentryProjects": sentry_projects,
-                "nextButton": {"allowedDomain": "https://vercel.com", "text": _("To Vercel")},
+                "nextButton": {
+                    "allowedDomain": "https://vercel.com",
+                    "description": _(
+                        "Link your Sentry projects to complete your installation on Vercel"
+                    ),
+                    "text": _("Complete on Vercel"),
```

Comment:
@EvanPurkhiser might make more sense to say `Complete Installation` here

Reply:
I like that it makes it obvious that you're going _back_ to vercel though. Maybe there's some wording so we can convey both?

Reply:
@EvanPurkhiser True. How about `Complete & Return to Vercel`?